spec/models/king_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe King, type: :model do
+  describe '#create' do
+    it 'creates a king in the database' do
+      king = FactoryGirl.create(:king)
+      expect(king.type == 'King').to eq(true)
+    end
+  end
+
+  describe '#valid_move?' do
+    it 'returns true if the move is diagonal up-right one square' do
+      king = FactoryGirl.create(:king, x_position: 4, y_position: 4)

```

Comment:
Uhhh . . . conflict markers? I had never even heard of 'em until today, and I haven't put any of them in my local files.